### DISCUSSION TITLE
looking for CBS Reality

### DISCUSSION THREAD

**Discussion by veso266** [iptv-org/community#1106] [Q&A]
Hi there, I found out my favourite channel CBS Reality: https://www.cbsreality.tv/eu/

closed this year so I cannot watch it anymore via my TV Provider (Telekom Slovenija)

I heared the channel is still available in Romania, Greece, Gibraltar and Poland
Anyone knows where can I find the stream for it (preferably in English, hope its not dubbed in there :smile: )

Thanks

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
